README.md

```diff
@@ -24,131 +25,183 @@ At present time this open-source repository is maintained separately from the Ca
+
+Cache Server supports two caching mechanisms:
+
+* A file system based cache.
+* A fully memory (RAM) backed cache.
+
+The file system module is the default and suitable for most applications. The RAM cache module provides optimal performance but requires a sufficient amount of physical RAM in the server system. Typically, this is two to three times size of your Project's Library folder on disk.
 
 Configuration options for all modules are set in the `config/default.yml` file.
+
 ### cache_fs (default)
+
 A simple, efficient file system backed cache.
+
 #### Usage
+
 `--cache-module cache_fs`
+
 #### Options
+
 Option                          | Default     | Description
 ------------------------------- | ----------- | -----------
-cachePath                       | `.cache_fs` | Path to cache directory
-cleanupOptions.expireTimeSpan   | `P30D`      | [ASP.NET](https://msdn.microsoft.com/en-us/library/se73z7b9(v=vs.110).aspx) or [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601#Time_intervals) style timespan. Cache files that have not been accessed within this timespan will be eligible for cleanup/removal. The [moment](https://momentjs.com/docs/#/durations/) library is used to parse durations - more information on duration syntax can be found in the library documentation.
-cleanupOptions.maxCacheSize     | 0           | Size in bytes to limit overall cache disk utilization. The cleanup script will consider files for removal in least-recently-used order to bring the total disk utilization under this threshold. 0 disables this cleanup feature. 
-
-#### Notes
-* This module is backwards compatible with v5.x Cache Server directories
-* Supports worker threads (`--workers` option)
-### cache_ram
+cachePath                       | `.cache_fs` | The path to the cache directory.
+cleanupOptions.expireTimeSpan   | `P30D`      | [ASP.NET](https://msdn.microsoft.com/en-us/library/se73z7b9(v=vs.110).aspx) or [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601#Time_intervals) style timespan. Cache files that have not been accessed within this timespan are eligible for cleanup. For more information on duration syntax, see the [moment](https://momentjs.com/docs/#/durations/) library documentation.
+cleanupOptions.maxCacheSize     | 0           | The maximum size, in bytes, of the cache on disk. To bring the total disk utilization under this threshold, the cleanup script considers files for removal in least recently used order. Set the value to zero (0) to disable the cleanup feature.
+
+### Notes
+
+* cache_fs is backwards compatible with v5.x Cache Server directories.
+* Supports worker threads using the `--workers` option.
+* When you run the cleanup script, the value of the `expireTimeSpan` option is used to determine which file to delete files. If `maxCacheSize` is specified the script checks whether the cache exceeds the value of `maxCacheSize`. If it does, the script deletes files in least-recently-used order until the cache no longer exceed maxCacheSize.
```

Comment:
"which file to delete files" doesn't make sense

Reply:
Fixed.